rllib/algorithms/algorithm_config.py

```diff
@@ -3105,7 +3106,7 @@ def get_marl_module_spec(
         self,
         *,
         policy_dict: Dict[str, PolicySpec],
-        module_spec: Optional[SingleAgentRLModuleSpec] = None,
+        single_agent_rl_module_spec: Optional[SingleAgentRLModuleSpec] = None,
```

Comment:
Make this 100% explicit. There is some confusion when working with these specs as to which type is expected in which situation.